README.md

```diff
@@ -8,6 +8,13 @@ Callback when you reach the bottom of the page and ajax requests finish. Useful
 $.onInfiniteScroll(function() {
     // Fetch & append some content
 });
+
+$.onInfiniteScroll(function() {
+    // Pass an optional offset from the page bottom at which to trigger the callback.
+}, 100);
```

Comment:
I would actually prefer this be an options object so it has room to expand.

````
$.onInfiniteScroll(function() {
  // Pass an optional offset from the page bottom at which to trigger the callback.
}, { offset: 100 });
````